packages/devtools_app/lib/src/shared/screen.dart

```diff
@@ -145,6 +145,17 @@ enum ScreenMetaData {
 }
 
 /// Defines a page shown in the DevTools [TabBar].
+///
+/// A devtools screen can be in three modes: offline-data, connected
+/// and not-connected. The screen can support only subset of modes.
```

Comment:
what do you mean by "The screen can support only subset of modes"?

Reply:
Not all screens support all modes :)

Reply:
got it. Maybe rephrase for clarity? "A screen may support any combination of modes."